e2e/cloud_manager/helper_test.go

```diff
@@ -151,26 +167,26 @@ func generateRSConfig(filename, hostname, clusterName, version, fcVersion string
 		},
 	}
 
-	jsonEncoder := json.NewEncoder(feedFile)
+	jsonEncoder := json.NewEncoder(configFile)
 	jsonEncoder.SetIndent("", "  ")
-	return jsonEncoder.Encode(downloadArchive)
+	return jsonEncoder.Encode(cluster)
 }
 
 func generateShardedConfig(filename, hostname, clusterName, version, fcVersion string) error {
-	feedFile, err := os.Create(filename)
+	configFile, err := os.Create(filename)
 	if err != nil {
 		return err
 	}
-	defer feedFile.Close()
+	defer configFile.Close()
 
-	downloadArchive := &convert.ClusterConfig{
+	cluster := &convert.ClusterConfig{
 		RSConfig: convert.RSConfig{
 			FeatureCompatibilityVersion: fcVersion,
 			Name:                        clusterName,
 			Version:                     version,
 		},
 		Config: &convert.RSConfig{
-			Name: "configRS",
+			Name: clusterName + "_configRS",
```

Comment:
flaky, if a previous test failed to remove the cluster we would get duplicated name error here